tests/node/test_wakunode_store.nim

```diff
@@ -328,46 +346,53 @@ suite "Waku Store - End to End - Sorted Archive":
         # Then the response contains the archive.DefaultPageSize messages
         check:
           queryResponse1.get().messages == totalMessages[0..<archive.DefaultPageSize]
-        
+
         # Given the next query (2/2)
-        let historyQuery2 = HistoryQuery(
-          cursor: queryResponse1.get().cursor,
-          pubsubTopic: some(pubsubTopic),
-          contentTopics: contentTopicSeq,
-          direction: true,
-          pageSize: 0
-        )
+        let
+          historyQuery2 =
+            HistoryQuery(
+              cursor: queryResponse1.get().cursor,
+              pubsubTopic: some(pubsubTopic),
+              contentTopics: contentTopicSeq,
+              direction: PagingDirection.FORWARD,
+              pageSize: 0,
+            )
 
         # When making the next history query
         let queryResponse2 = await client.query(historyQuery2, serverRemotePeerInfo)
 
         # Then the response contains the remaining messages
         check:
-          queryResponse2.get().messages == totalMessages[archive.DefaultPageSize..<archive.DefaultPageSize + 5]
+          queryResponse2.get().messages ==
+            totalMessages[archive.DefaultPageSize..<archive.DefaultPageSize + 5]
 
       asyncTest "Pagination with Default Page Size":
         # Given a message list of size higher than the default page size
         let currentStoreLen = uint((await archiveDriver.getMessagesCount()).get())
-        assert archive.DefaultPageSize > currentStoreLen, "This test requires a store with more than (DefaultPageSize) messages"
+        assert archive.DefaultPageSize > currentStoreLen,
+          "This test requires a store with more than (DefaultPageSize) messages"
         let missingMessagesAmount = archive.DefaultPageSize - currentStoreLen + 5
 
         let lastMessageTimestamp = archiveMessages[archiveMessages.len - 1].timestamp
         var extraMessages: seq[WakuMessage] = @[]
         for i in 0..<missingMessagesAmount:
-          let 
-            timestampOffset = 10 * int(i + 1) # + 1 to avoid collision with existing messages
-            message: WakuMessage = fakeWakuMessage(@[byte i], ts=ts(timestampOffset, lastMessageTimestamp))
+          let
+            timestampOffset = 10 * int(i + 1)
+              # + 1 to avoid collision with existing messages
```

Comment:
If possible let's move this comment to the correct line, if the formatted doesn't allow it then all good :))

Reply:
That was the autoformatter, yes. It annoyed me too 😂